src/Connections/JiraProject.ts

```diff
@@ -1,5 +1,5 @@
-import { IConnection, IConnectionState } from "./IConnection";
-import { Appservice } from "matrix-bot-sdk";
+import { Connection, IConnection, IConnectionState, InstantiateConnectionOpts, ProvisionConnectionOpts } from "./IConnection";
```

Comment:
For simplicity while we evaluate this idea, I've not committed the changes I've made to other repos.